integration/log.go

```diff
@@ -363,7 +364,7 @@ func readbackLogEntries(logID int64, client trillian.TrillianLogClient, params T
 	return leafMap, nil
 }
 
-func checkLogRootHashMatches(tree *merkle.InMemoryMerkleTree, client trillian.TrillianLogClient, params TestParameters) error {
+func checkLogRootHashMatches(tree *memory.InMemoryMerkleTree, client trillian.TrillianLogClient, params TestParameters) error {
```

Comment:
Since you are moving it anyway (breaking change), consider renaming the package/type to `inmemory.MerkleTree`.

Reply:
done